content/src/view.js

```diff
@@ -245,16 +250,21 @@ function initialPaneState() {
 }
 
 function setOnCallback(tag, cb) {
-  state.callbacks[tag] = cb;
+  if (state.callbacks[tag] == null){
+    state.callbacks[tag] = [];
+  }
+  state.callbacks[tag].push(cb);
 }
 
 function fireEvent(tag, args) {
   if (tag in state.callbacks) {
-    var cb = state.callbacks[tag];
-    if (cb) {
-      cb.apply(null, args);
-    }
-  }
+    var cbs = state.callbacks[tag].slice();
+    for (j=0; j < cbs.length; j++ ){
```

Comment:
Space around ` = `, no space before `)` and space between `) {`.

Thanks for your help.  Consistent spacing makes it easier to grep through code.